views/pages/login.handlebars

```diff
@@ -0,0 +1,34 @@
+<div class="row">
+    <div class="col-sm-12">
+        <form id="login-form" method="POST" action="/users/login">
+
+            <div class="form-group">
+                <label for="email">Email</label>
+                <input type="email" placeholder="Enter Email" name="email" id="email" value="{{email}}"
+                    placeholder="email" required>
+            </div>
+
+            <div class="form-group">
+                <label for="password">Password</label>
+                <input type="password" placeholder="Enter Password" name="password" id="password" value="{{password}}"
+                    placeholder="password" required>
+            </div>
+
+            {{#if error }}
+            <div class="form-group">
+                <div class="alert alert-danger">
+                    {{ error }}
+                </div>
+            </div>
+            {{/if}}

```

Comment:
I think 2nd error should be {{ errorInfo }}, otherwise it will boolean 'true' on screen, when people trigger this error